src/sidebar/components/VersionInfo.js

```diff
@@ -9,6 +10,35 @@ import { withServices } from '../service-context';
  * @prop {import('../services/toast-messenger').ToastMessengerService} toastMessenger
  */
 
+/**
+ * @typedef VersionEntryProps
+ * @prop {boolean} [breakAll=false] - Should this entry break long strings of
+ *   text, even if they are mid-word? (This prevents long strings from breaking
+ *   layout but may cause awkward breaks in shorter text)
+ * @prop {string} field
+ * @prop {string} value
+ */
+
+/**
+ * Render an entry for one piece of version data.
+ *
+ * @param {VersionEntryProps} props
+ */
+function VersionEntry({ breakAll = false, field, value }) {
```

Comment:
Refactoring this component to use the lowest-level utility classes possible helped me spot that this little local subcomponent could be extracted. I imagine that lots of little components are corollary to using a utility-first approach, and I don't mind it.